lms/envs/aws.py

```diff
@@ -434,6 +434,10 @@
 # within the same S3 bucket.
 ORA2_FILE_PREFIX = ENV_TOKENS.get("ORA2_FILE_PREFIX", ORA2_FILE_PREFIX)
 
+### Course forums usage download
+COURSE_FORUMS_USAGE_DOWNLOAD = ENV_TOKENS.get("COURSE_FORUMS_USAGE_DOWNLOAD", COURSE_FORUMS_USAGE_DOWNLOAD)
```

Comment:
@caijim ENV_TOKENS are defined in lms.env.json file in the parent directory of edx-platform